packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -188,6 +188,15 @@ export class PKPBase<T = PKPBaseDefaultParams> {
     }
   }
 
+  /**
+   * Return address
+   *
+   * @returns {Promise<string>} - Address associated with concrete type of PKPBase
+   */
+  getAddress(): Promise<string> {
+    return Promise.resolve('');
```

Comment:
should we mark some TODO here?

Reply:
or is this meant to be overwritten? Can we not mark just this function as `abstract` then?

Reply:
or we can use a mixture of your class inheritance here and also an interface method that the subclasses off of `PKPBase` would `implements`

Reply:
added comment about subclasses and changed to throw an error instead